src/rpcserver.cpp

```diff
@@ -368,6 +368,16 @@ const CRPCCommand *CRPCTable::operator[](string name) const
     return (*it).second;
 }
 
+void CRPCTable::AddOrReplaceCommand(const CRPCCommand command)
+{
+    // search after existing key in hashmap and remove it
+    map<string, const CRPCCommand*>::iterator it = mapCommands.find(command.name);
```

Comment:
Why is this lookup needed? You can just overwrite using the line you have below?